### PR TITLE
xbps-pkgdb(1), xbps-install(1), xbps-remove(1): extend discussion of package modes

### PR DESCRIPTION
**Pull request by roman-neuhauser**
description of implications of the various modes was scattered across the man pages
and not explicit enough for my taste.

### CODE REVIEW

File: bin/xbps-pkgdb/xbps-pkgdb.1 [hunk 1]
```diff
@@ -48,23 +48,51 @@ Enables extra debugging shown to stderr.
...
+.It Sy repolock , repounlock
+Set or unset the
+.Sy repolock
+mode for each
+.Ar PKGNAME .
 A package in
 .Sy repolock
 mode will only accept updates that are available in the same repository that was used
 for installing.
+.\" FIXME: no way to list repolocked packages?
```
Comment (ebfe) [inline bin/xbps-pkgdb/xbps-pkgdb.1]: xbps-query -p repolock -s yes

Comment (roman-neuhauser) [inline bin/xbps-pkgdb/xbps-pkgdb.1]: what are the available properties?  i found a list in the zsh completion file (data/_xbps) but would prefer something more authoritative.  i tried to follow the code, the rabbit hole seems to end in lib/portableproplib/prop_dictionary.c without a clear answer.

Comment (roman-neuhauser) [inline bin/xbps-pkgdb/xbps-pkgdb.1]: also, what exactly does the `-s yes` part do?

    % xbps-query -p repolock -s yes   
    bash-4.4.0_1: true
    % xbps-query -p repolock -s no 
    bash-4.4.0_1: true

**Comment by roman-neuhauser** [comment]
i replaced the FIXME comment with a suggestion to use `xbps-query -p repolock ...`  and rebased the branch.

File: bin/xbps-pkgdb/xbps-pkgdb.1 [hunk 2]
```diff
@@ -48,23 +48,55 @@ Enables extra debugging shown to stderr.
...
-and repository locked mode.
-A package that was installed as dependency will be in
+.
+.Bl -tag -width -x
+.
+.It Sy auto , manual
+.Sy auto No marks each Ar PKGNAME
+as
+.Sy automatic -
+as if it was installed implicitely as a dependency of another package.
```
Comment (ebfe) [inline bin/xbps-pkgdb/xbps-pkgdb.1]: implicitly?

File: bin/xbps-pkgdb/xbps-pkgdb.1 [hunk 3]
```diff
@@ -48,23 +48,55 @@ Enables extra debugging shown to stderr.
...
+.Sy manual No marks each Ar PKGNAME
+as if it was installed explicitely as an operand to
```
Comment (ebfe) [inline bin/xbps-pkgdb/xbps-pkgdb.1]: explicitly?

**Comment by roman-neuhauser** [comment]
rebased with spelling fixes and `xbps-query --list-repolock-pkgs` mention (#202)

**Pull request merged**